screens/TimeSelectorScreen.js

```diff
@@ -1,72 +1,94 @@
-import { StyleSheet, Text, View, Picker } from "react-native";
+import { StyleSheet, Text, View, Picker, Platform, Image } from "react-native";
 import Button from "react-native-button";
 import moment from "moment";
 
 import { writeToStorage, readFromStorage, utilizeStartDate } from "../utils";
 
 import { TASKS } from "../constants/Tasks";
 
+
 import AppStyle from "../AppStyle";
+import Colors from "../constants/Colors";
 
 export default function TimeSelectorScreen({
   navigation,
   route: { params: { selectedStartDate } } = {}
 }) {
-  const [day, setDay] = useState("14");
 
-  useEffect(() => {}, []);
+  useEffect(() => { }, []);
+  const calculateStartDate = (date) => {
+    var now = moment();
+    date.hour(now.hour());
+    date.minute(now.minute());
+    date.second(now.second());
+  }
+  console.log('selectedDate: ', selectedStartDate);
+
+  const [day, setDay] = useState('14');
+
+
+
 
   const days = [];
 
   for (var i = 1; i <= 30; i++) {
     days.push(i);
   }
 
+
   return (
     <View style={styles.container}>
       <Text style={styles.header}>Choose Duration</Text>
-      <Picker
-        selectedValue={day}
-        onValueChange={val => {
-          setDay(val);
-        }}
-      >
-        {days.map((item, index) => {
-          return (
-            <Picker.Item
-              label={item.toString()}
-              value={item.toString()}
-              key={item}
-            />
-          );
-        })}
-      </Picker>
-      <View style={styles.buttonContainer}>
-        <Button
-          style={styles.button}
-          onPress={async () => {
-            await writeToStorage("quarantineDurationInDays", day);
-            // utilizeStartDate(JSON.parse(date));
-            await writeToStorage(
-              "quarantineStartDate",
-              utilizeStartDate(JSON.parse(selectedStartDate))
-            );
-            await writeToStorage("tasks", TASKS.slice(0, day));
-
-            navigation.reset({
-              index: 0,
-              routes: [{ name: "App" }]
-            });
-          }}
-        >
-          Start
-        </Button>
+      {Platform.OS === 'android' &&
+
+        <Image
+          style={styles.image}
+          source={require('../assets/images/time-selector.png')}
+        />
+      }
+      <View style={Platform.OS === 'android' ? styles.buttonContainer : { flex: 1 }}>
+        <View style={Platform.OS === 'android' ? styles.pickerWrapper : {}}>
+          <Picker
+            selectedValue={day}
+            onValueChange={(val) => { setDay(val) }}
+            style={Platform.OS === 'android' ? styles.picker : {}}
+            prompt="Duration"
+          >
+            {days.map((item, index) => {
+              return (
+                <Picker.Item label={item.toString()} value={item.toString()} key={item} />
+              );
+            })}
+
+          </Picker>
+        </View>
+        <View style={Platform.OS === 'ios' ? styles.buttonContainer : {}}>
```

Comment:
`Platform.OS === 'ios' ? styles.buttonContainer : {}`
insted of this you can use like;

`condition && yourVariable`